django_irods/utils.py

```diff
@@ -0,0 +1,39 @@
+import binascii
+from django.db import connection
+
+
+def bucket_and_name(path):
+    res_id = "/".join(path.split("/")[:1])
```

Comment:
no, it creates a circular dependency. hs_core uses this storage layer so we can't have the storage layer use hs_core. Of course I initially tried that and then had to write the raw sql queries because of the circular dependency.